cognite/client/_api/sequences.py

```diff
@@ -0,0 +1,403 @@
+            Union[Sequence, SequenceList]: Updated sequences.
+
+        Examples:
+
+            Update a sequences that you have fetched. This will perform a full update of the sequences::
+
+                >>> from cognite.client.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> res = c.sequences.retrieve(id=1)
+                >>> res.description = "New description"
+                >>> res = c.sequences.update(res)
+
+            Perform a partial update on a sequences, updating the description and adding a new field to metadata::
+
+                >>> from cognite.client.experimental import CogniteClient
+                >>> from cognite.client.data_classes import SequenceUpdate
+                >>> c = CogniteClient()
+                >>> my_update = SequenceUpdate(id=1).description.set("New description").metadata.add({"key": "value"})
+                >>> res = c.sequences.update(my_update)
+        """
+        return self._update_multiple(items=item)
+
+    def search(
+        self,
+        name: str = None,
+        description: str = None,
+        query: str = None,
+        filter: Union[SequenceFilter, Dict] = None,
+        limit: int = None,
+    ) -> SequenceList:
+        """Search for sequences.
+
+        Args:
+            name (str, optional): Prefix and fuzzy search on name.
+            description (str, optional): Prefix and fuzzy search on description.
+            query (str, optional): Search on name and description using wildcard search on each of the words (separated
+                by spaces). Retrieves results where at least one word must match. Example: 'some other'
+            filter (Union[SequenceFilter, Dict], optional): Filter to apply. Performs exact match on these fields.
+            limit (int, optional): Max number of results to return.
+
+        Returns:
+            SequenceList: List of requested sequences.
+
+        Examples:
+
+            Search for a sequences::
+
+                >>> from cognite.client.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> res = c.sequences.search(name="some name")
+        """
+        return self._search(
+            search={"name": name, "description": description, "query": query}, filter=filter, limit=limit
+        )
+
+
+class SequencesDataAPI(APIClient):
+    _DATA_PATH = "/sequences/data"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._SEQ_POST_LIMIT = 10000
+
+    def insert(
+        self,
+        columns: List[Union[int, str]],
+        rows: Dict[int, List[Union[int, float, str]]],
+        id: int = None,
+        external_id: str = None,
+    ) -> None:
+        """Insert rows into a sequence
+
+        Args:
+            columns: list of id or external id for the columns of the sequence
+            rows: dictionary of row number => data, where data is a list corresponding to the given columns
```

Comment:
Missing type for columns and rows. First letter of description should be capitalized.